iac-extensions/docker/src/test/resources/checks/ExecutableNotOwnedByRootCheck/add_copy.dockerfile

```diff
@@ -0,0 +1,99 @@
+FROM scratch
+
+ARG SENSITIVE_USER=other
+ARG COMPLIANT_USER=root
+ARG SENSITIVE_CHMOD=u+w
+ARG COMPLIANT_CHMOD=u+r
+ARG SENSITIVE_FILE=file.sh
+ARG COMPLIANT_FILE=file.txt
+
+## All possible basic use cases
+ADD --chown=root               file.sh  target/
+ADD --chown=root               file.txt target/
+ADD --chown=root  --chmod=u+x  file.sh  target/
+ADD --chown=root  --chmod=u+x  file.txt target/
+ADD --chown=root  --chmod=u+w  file.sh  target/
+ADD --chown=root  --chmod=u+w  file.txt target/
+ADD --chown=root  --chmod=u+wx file.sh  target/
+ADD --chown=root  --chmod=u+wx file.txt target/
+# Noncompliant@+1
+ADD --chown=other              file.sh  target/
+ADD --chown=other              file.txt target/
+ADD --chown=other --chmod=u+x  file.sh  target/
+ADD --chown=other --chmod=u+x  file.txt target/
+# Noncompliant@+1
+ADD --chown=other --chmod=u+w  file.sh  target/
+ADD --chown=other --chmod=u+w  file.txt target/
+# Noncompliant@+1
+ADD --chown=other --chmod=u+wx file.sh  target/
+# Noncompliant@+1
+ADD --chown=other --chmod=u+wx file.txt target/
```

Comment:
I like the column formatting

Reply:
There are so much different use cases and the logic to differentiate compliant/sensitive is not straightforward.
There was a real need to be able to compare those cases to better understand which ones are sensitive and which ones are compliant.
Happy that you liked it :D